packages/bpk-docs/src/pages/NativeBadgePage/NativeBadgePage.js

```diff
@@ -51,6 +53,28 @@ const components = [
       },
     ],
   },
+  {
+    id: 'withAccessoryView',
```

Comment:
Convention for IDs is kebab case: `with-accessory-view`.

Reply:
The link in `unreleased.md` will need to change accordingly too.

Reply:
Great catch for `UNRELEASED.md`